Develop - Calculator logic and styles corrected

Kruchy1980 opened the pull request:
Tutaj chyba po polsku czy się mylę??
Do kalkulatora poprawki logiki zrobione i drobne css'y  logika działa mam nadzieję że już tak jak chciałeś w związku z czym chcę spullować na mastera a nie na developa. Pozdrawiam  i czekamy na odpowiedź

Kinga-Zawarczynska committed:
changed both files

Kinga-Zawarczynska committed:
resolved problems in style

Commit:
nil

Commit:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into Filip-Siwiec

Commit:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into Filip-Siwiec

Commit:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into Filip-Siwiec

Commit:
change mobile.class names (english)

Commit:
Merge branch 'Filip-Siwiec' into develop

Kruchy1980 committed:
smaller picture fo thumb added

Kruchy1980 committed:
Image for slider stylized

Kruchy1980 committed:
third image for slider thumb

Kruchy1980 committed:
Calculator large style corrected

AdamToczko committed:
navka zmienione logo w html
mobile - wyrzucony header , logo linkuje, zmieniony kolor hamburgerana ten sam co logo

Kruchy1980 committed:
mobile calculator - style initial

AdamToczko committed:
merged with origin develop

Kruchy1980 committed:
New version of calculator initial

AdamToczko committed:
merged with AdamToczko changes to nav

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

AdamToczko committed:
further work on navbar

AdamToczko committed:
Merge branch 'develop' into AdamToczko

AdamToczko committed:
further work on responsiveness
mobile.css finished
tablet css finished
working on medium
and style = large
removed in index food waste link from navigation as logo will be top link

Kruchy1980 committed:
style of calculator done

Kruchy1980 committed:
mobile version of style of  calculator finished

AdamToczko committed:
finished mobile, tablet, medium, style(large)
addedd styling for calculator to mobile and to index as well

AdamToczko committed:
Merge branch 'AdamToczko' into develop

Kruchy1980 committed:
Calculator styles finished

AdamToczko committed:
started reworking  team section

Kruchy1980 committed:
Calculator logic fixed plus
calculation function on move added0n the slider move and automatical change the Subscription time, plus plus communicate-alert when no input checked added

Kruchy1980 committed:
Small corrections added to calculator4.js

Commit:
cookie banner/storage in progress, game instruction-in progress

Commit:
merge

Commit:
Merge branch 'develop' into Filip-Siwiec

AdamToczko committed:
finish mobile/tablet/medium/styles/team

AdamToczko committed:
Merge branch 'origin develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

Conflicts:
	style.css
 only on cookie banner resolved

AdamToczko committed:
Merge branch 'AdamToczko' into develop

Kruchy1980 committed:
Merge remote-tracking branch 'origin' into jk-fw-land-p

AdamToczko committed:
Merge branch 'develop' into AdamToczko

Kruchy1980 committed:
Logic and information to calcualtor added and styles as well

Kruchy1980 committed:
chilii-thumb-.png removed

Kruchy1980 committed:
small correction to slider thumb background added

Kruchy1980 committed:
Styles added and connected, calculator corrected

Kinga-Zawarczynska committed:
working button start game, just one click possible

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

Kruchy1980 committed:
Unnecessary files of calculator removed

Kinga-Zawarczynska committed:
character added to the game

Kruchy1980 committed:
Merge branch 'jk-fw-land-p' into develop

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+const input = document.querySelectorAll(".months");
+// Range Slider full module.
+const rangeslider = document.getElementById("sliderRange");
+let output = document.getElementById("quantity");
```
const

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+// Discount
+let discount = 0;
+
+let calculationOutput = document.querySelector(".calculation-output");
```
const

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+const rangeslider = document.getElementById("sliderRange");
+let output = document.getElementById("quantity");
+output.innerHTML = rangeslider.value;
+rangeslider.oninput = function() {
```
zaleca się  stosowanie addEventListener

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+//-------------------------------------------------------
+//Functions
+// Account Quantity from slider
+let getAccQuantity = () => {
```
nadmiarowy kod.. uzyte tylko raz, rownie dobrze mógłbyś od razu przypisać tą wartość document.getElementById("quantity").innerText we własciwej linijce

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+// Main full function of calculator
+
+function priceCalculation() {
+  accQuantity = getAccQuantity();
```
brak deklaracji zmiennej
dodatkowo od razu mozna by bylo tu poprawc wartosc document.getElementById("quantity").innerText

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+function priceCalculation() {
+  accQuantity = getAccQuantity();
+  timeOfSubscription = getTimeOfSubscription();
+  if (accQuantity >= 1 && accQuantity <= 4) {
```
po co ten if, skoro w środku masz kolejne ify ?

aszablowski replied:
dalej dziwna ifologia, kod skomplikowany. jeszcze raz odwołuje do przykładowego kodu który ma  dokładnie 20 linijek z komentarzami

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+        finalPrice.toFixed(2) + "PLN za konto, zniżka " + discount + " %");
+    } else if (timeOfSubscription == 2) {
+      discount = 5;
+      finalPrice = basePrice * (1 - discount / 100);
```
kod sie powrarza w kazdym ifie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+    } else if (timeOfSubscription == 2) {
+      discount = 5;
+      finalPrice = basePrice * (1 - discount / 100);
+      return (calculationOutput.innerHTML =
```
kod sie powtarza w kazdym ifie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+        finalPrice.toFixed(2) + "PLN za konto, zniżka " + discount + " %");
+    } else if (timeOfSubscription == 3) {
+      discount = 10;
+      finalPrice = basePrice * (1 - discount / 100);
```
kod sie powrarza w kazdym ifie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+    } else if (timeOfSubscription == 3) {
+      discount = 10;
+      finalPrice = basePrice * (1 - discount / 100);
+      return (calculationOutput.innerHTML =
```
kod sie powrarza w kazdym ifie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  } else if (accQuantity >= 5 && accQuantity <= 9) {
+    discount = 5;
+    if (timeOfSubscription == 1) {
+      finalPrice = basePrice * (1 - discount / 100);
```
kod sie powrarza w kazdym ifie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+    discount = 5;
+    if (timeOfSubscription == 1) {
+      finalPrice = basePrice * (1 - discount / 100);
+      return (calculationOutput.innerHTML =
```
kod sie powrarza w kazdym ifie

aszablowski commented on caculator.html:
```diff
@@ -1,76 +0,0 @@
-            <input
-              type="radio"
-              name="months"
-              id="months"
```
nie moze kilka elementów miec takich samych id, id musi byc unikalne

aszablowski commented on caculator.html:
```diff
@@ -1,76 +0,0 @@
-            <input
-              type="radio"
-              name="months"
-              id="months"
```
nie moze kilka elementów miec takich samych id, id musi byc unikalne

aszablowski commented on caculator.html:
```diff
@@ -1,76 +0,0 @@
-            <input
-              type="radio"
-              name="months"
-              id="months"
```
nie moze kilka elementów miec takich samych id, id musi byc unikalne

aszablowski commented on caculator.html:
```diff
@@ -1,76 +0,0 @@
-            <input
-              type="radio"
-              name="months"
-              id="months"
```
nie moze kilka elementów miec takich samych id, id musi byc unikalne

aszablowski commented on index.html:
```diff
@@ -291,17 +298,17 @@ <h2>Nasz zespół:</h2>
-              <label for="fourth-year">powyżej 36</label>
-              <input type="radio" name="months" id="months" class="months-fourth" />
+              <label for="months-first">do 12</label>
+              <input type="radio" name="months" id="months" class="months months-first"/>
```
id musi byc unikalne.. poza tym chyba tego id nie uzywasz?

aszablowski commented on index.html:
```diff
@@ -291,17 +298,17 @@ <h2>Nasz zespół:</h2>
+              <label for="months-first">do 12</label>
+              <input type="radio" name="months" id="months" class="months months-first"/>
+              <label for="months-second">do 24</label>
+              <input type="radio" name="months" id="months" class=" months months-second" />
```
id musi byc unikalne.. poza tym chyba tego id nie uzywasz?

aszablowski commented on index.html:
```diff
@@ -291,17 +298,17 @@ <h2>Nasz zespół:</h2>
+              <label for="months-second">do 24</label>
+              <input type="radio" name="months" id="months" class=" months months-second" />
+              <label for="months-third">do 36</label>
+              <input type="radio" name="months" id="months" class="months months-third" />
```
id musi byc unikalne.. poza tym chyba tego id nie uzywasz?

aszablowski commented on index.html:
```diff
@@ -291,17 +298,17 @@ <h2>Nasz zespół:</h2>
+              <label for="months-third">do 36</label>
+              <input type="radio" name="months" id="months" class="months months-third" />
+              <label for="months-fourth">powyżej 36</label>
+              <input type="radio" name="months" id="months" class=" months months-fourth" />
```
id musi byc unikalne.. poza tym chyba tego id nie uzywasz?

Kinga-Zawarczynska committed:
background and fruits added to the game

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  }
+}
+input.forEach(i => {
+  i.addEventListener("click", () => {
```
radiobuttony naklada sie event na "change" a nie click

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+// Get time of Subsccriptions function
+let getTimeOfSubscription = () => {
+  if (input[0].checked) {
+    return (timeOfSubscription = 1);
```
wystarczy return 1

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  if (input[0].checked) {
+    return (timeOfSubscription = 1);
+  } else if (input[1].checked) {
+    return (timeOfSubscription = 2);
```
wystarczy return 2

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  } else if (input[1].checked) {
+    return (timeOfSubscription = 2);
+  } else if (input[2].checked) {
+    return (timeOfSubscription = 3);
```
wystarczy return 3

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  } else if (input[2].checked) {
+    return (timeOfSubscription = 3);
+  } else if (input[3].checked) {
+    return (timeOfSubscription = 4);
```
wystarczy return 4

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,150 @@
+  } else if (input[3].checked) {
+    return (timeOfSubscription = 4);
+  } else {
+    alert("Musisz zaznaczyć długość subskrypcji");
```
alert ble.. zaznacz domyslnie jedna wartosc.. np minimalna

aszablowski commented on cookies.js:
```diff
@@ -1,7 +1,40 @@
+    document.cookie ="name=user;expires=Fri, 31 Dec 9999 23:59:59 GMT;path=/;SameSite=Lax";
+}
+
 guzik.addEventListener('click',function(){
```
brakuje odczytania cookie.. przez co cały czas pokazuje sie komunikat o cookie

aszablowski commented on cookies.js:
```diff
@@ -1,7 +1,40 @@
+
+
+
+// const Wyswietl = () => {
```
szkoda ze ten kod jest zakomentowany.. bo by moze działały cookie prawidłowo ;)

aszablowski commented on game-script.js:
```diff
@@ -123,10 +123,15 @@ const player1 = new Player(domRectagle);
+var newFruitInterval;
+clearInterval(falling);
+  clearInterval(newFruitInterval);
+  console.log(clearInterval(falling));
```
console.log na mastera nie powinny wchodzic

aszablowski commented on game-script.js:
```diff
@@ -123,10 +123,15 @@ const player1 = new Player(domRectagle);
 
-let newFriut;
-let falling;
+var falling;
```
var nie uzywamy, let albo const

aszablowski commented on game-script.js:
```diff
@@ -123,10 +123,15 @@ const player1 = new Player(domRectagle);
-let newFriut;
-let falling;
+var falling;
+var newFruitInterval;
```
var nie uzywamy, let albo const

aszablowski commented on index.html:
```diff
@@ -193,12 +193,14 @@ <h4>Dodaj komentarz</h4>
 
-  </div>
+  
   </form>
```
formatowanie, brak tabulacji

aszablowski commented on index.html:
```diff
@@ -193,12 +193,14 @@ <h4>Dodaj komentarz</h4>
+                <div class="calculator-button-box"><a href="#" class="trigger">Kalkulator subskrypcji !</a></div>
           <p class="news-small rodo"> <a href="rodo.html">Twoje dane są z nami bezpieczne!</a> </p>
         </div>
 
```
niepotrzebne entery

aszablowski commented on index.html:
```diff
@@ -343,6 +374,21 @@ <h2>Nasz zespół:</h2>
+// x.addListener(myFunction2) // Attach listener function on state changes
+
 
 var slideIndex = 1;
```
var czy let?

aszablowski commented on index.html:
```diff
@@ -394,7 +440,7 @@ <h2>Nasz zespół:</h2>
 function autoSlides() {
-  console.log('autoSlides');
+  // console.log('autoSlides');
   var i;
```
var czy let?

aszablowski commented on index.html:
```diff
@@ -394,7 +440,7 @@ <h2>Nasz zespół:</h2>
-  console.log('autoSlides');
+  // console.log('autoSlides');
   var i;
   var slides = document.getElementsByClassName("mySlides");
```
var czy const?

Kruchy1980 committed:
Final - i hope version of logic of calculator fine

Kinga-Zawarczynska committed:
clearinterval in the game works

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into jk-fw-land-p

AdamToczko committed:
worked on all below files, made navigation working with logo showing/notshowing
moved js for navigation to separate script

 On branch AdamToczko
 Changes to be committed:
	modified:   index.html
	modified:   medium.css
	modified:   mobile.css
	new file:   navigation.js
	deleted:    style-nav.css
	modified:   style.css
	modified:   tablet.css

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

AdamToczko committed:
deleted additional photos not used

AdamToczko committed:
Merge branch 'AdamToczko' into develop

AdamToczko committed:
moved slides js from htm to js file

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,91 @@
+  else if (input[3].checked) discount += 15;
+
+  finalPrice = basePrice * (1 - discount / 100);
+  console.log(finalPrice);
```
console.log

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,91 @@
+  else if (rangeslider.value >= 5 && rangeslider.value <= 9) discount = 5;
+  else if (rangeslider.value == 10) discount = 7;
+  // time of Subscription
+  if (input[0].checked) discount;
```
niepotrzebny if, nic nie robi

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,91 @@
+  });
+});
+
+function priceCalculation() {
```
dobrze wywołac tą funkcje od razu na starcie

aszablowski commented on calculator4.js:
```diff
@@ -0,0 +1,91 @@
+
+// Continue calculation in case of  radiobutton change
+input.forEach(i => {
+  i.addEventListener("click", () => {
```
click, lepiej change

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+
+
+var slideIndex = 1;
```
var, czy let?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+ wrapper.addEventListener("mouseleave", autoSlides);
+
+
+var timeOut;
```
var, czy let?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+  function manualSlides(n) {
+    // console.log('manualSlides');
+    clearTimeout(timeOut);
+    var i;
```
var, czy let?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+    // console.log('manualSlides');
+    clearTimeout(timeOut);
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
```
var, czy const?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+    clearTimeout(timeOut);
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    var dots = document.getElementsByClassName("dot");
```
var, czy const?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+
+
+
+  var autoSlideIndex = 0;
```
var, czy let?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+
+function autoSlides() {
+  // console.log('autoSlides');
+  var i;
```
var, czy let?

aszablowski commented on slides.js:
```diff
@@ -0,0 +1,64 @@
+function autoSlides() {
+  // console.log('autoSlides');
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
```
var, czy const

aszablowski commented on navigation.js:
```diff
@@ -0,0 +1,34 @@
+  const navbar = document.querySelector("#navbar");
+  const sticky = navbar.offsetTop;
+  function navigationBehavior() {
+  if (window.matchMedia("(min-width: 800px)") && window.pageYOffset >= sticky) {
```
w kodzie ponizej brakuje wcięć

AdamToczko committed:
added header to mobile and tablet

AdamToczko committed:
Merge branch 'AdamToczko' into develop

Kinga-Zawarczynska committed:
different fruits are falling

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

Kruchy1980 committed:
Mail validation initial

Kinga-Zawarczynska committed:
game over show works

Kinga-Zawarczynska committed:
erased unimportant comments

Kruchy1980 committed:
Email validation
added to game-instruction.js file and let the user run the instruction and game only if the name or validation email will be valid

Kruchy1980 committed:
Some Button styles added

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into jk-fw-land-p

Kruchy1980 committed:
Branches merged succesfully;)

Commit:
cookies almost finished

Commit:
cookies finished

Commit:
cookies fixed

Kruchy1980 committed:
Validation completed and style of calculator partially modified

Kruchy1980 committed:
modal calculator window styles corrected

Kruchy1980 committed:
Style of calculator modified to large size screen

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into jk-fw-land-p

Kruchy1980 committed:
Fotter style on mobile corrected

Commit:
fixed cookies mobile / deleted usused png

Commit:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into Filip-Siwiec

Kruchy1980 committed:
Calculator styles update - font-family added

Commit:
fixed cookies mobile

Kruchy1980 committed:
Calculator calculate prices immediately after is on

Kruchy1980 committed:
Price Calculator fixed

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into jk-fw-land-p

Kruchy1980 committed:
---

AdamToczko committed:
small changes, trying with navigation hoghlight

AdamToczko committed:
Merge branch 'develop' into AdamToczko

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

AdamToczko committed:
resolved conflicts from develop

Kruchy1980 committed:
Corrections to calculator

AdamToczko committed:
changed slides.js

Kruchy1980 committed:
Final version of calculator, bugs corrected
styles corected and js file changed to be better in index as well

Kruchy1980 committed:
Develop merged to my personal branch

AdamToczko committed:
no changes working on highliting sections in nav

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

Commit:
game instr. and responsivity

Commit:
game instr an respon

Commit:
instruction and cookies fixed

AdamToczko reviewed (commented):
tested all working

AdamToczko committed:
changed styles for newsleter form

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

Kinga-Zawarczynska committed:
erased some comments in slides and game scripts

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

AdamToczko committed:
nav links are highlighting on large style.css when scrolling through sections

AdamToczko committed:
further work onhigh light of nav/section

AdamToczko committed:
nav

AdamToczko committed:
Merge branch 'AdamToczko' into develop

Kinga-Zawarczynska committed:
game is working now, no highscore yet

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

Kinga-Zawarczynska committed:
fixed conflicts with develop

Kinga-Zawarczynska committed:
changed font-sizes to smaller ones

Kinga-Zawarczynska committed:
changed font-size in slides

Kinga-Zawarczynska committed:
changes the code in the game with comments from Adam

Commit:
instruction text update

Commit:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into Filip-Siwiec

Kruchy1980 committed:
Styles of calculator added

Commit:
instruction in game added

AdamToczko committed:
changed styles for buttons for all viewports
navigation.js - working on highlighting

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

AdamToczko committed:
further button styles changes

Kruchy1980 committed:
Small corrections to game instruction with name validation added

Kinga-Zawarczynska committed:
Merge branch 'develop' into Kinga

Commit:
finished instructions, small code cleanup

AdamToczko committed:
navigation highlights work on medium and large ans not applicable on small as nav links hidden
Changed styles for hero and some buttons
hero size need to be fixed and cannot be in vh as it will be changing all the time and affect all other sections
 and navigation js

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

 Conflicts:
	style.css

Kinga-Zawarczynska committed:
making highscore of the game

Kinga-Zawarczynska committed:
merged with Kinga

Kruchy1980 committed:
Game HTML conflicts resolved

AdamToczko committed:
small changes to styles

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

AdamToczko committed:
a

Kinga-Zawarczynska committed:
erased one class from game

AdamToczko committed:
removed console.log from nav.js

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

Kinga-Zawarczynska committed:
changed get Element to queryselector in line 210

Kruchy1980 committed:
NIL

Kruchy1980 committed:
Merge with develop

Kinga-Zawarczynska committed:
game works, points show

Kruchy1980 committed:
Style of game correction - not working yet

Kruchy1980 committed:
Game files modified and  working

Kruchy1980 committed:
NIL

AdamToczko committed:
modified:   index.html
        modified:   navigation.js
        modified:   style.css
so that navigation works

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

Kinga-Zawarczynska committed:
game works, highscore works

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

AdamToczko committed:
working on resolving nav highlight

Kruchy1980 committed:
Game modified

AdamToczko committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into AdamToczko

Kruchy1980 committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into jk-fw-land-p

Kruchy1980 committed:
Merged to develop

Kinga-Zawarczynska committed:
fruits are falling normally when clicking start again

Kinga-Zawarczynska committed:
Merge branch 'develop' of https://github.com/infoshareacademy/jfdd13-jastrzebienocy into develop

Kruchy1980 committed:
Validation corrected - once more:)

Kinga-Zawarczynska committed:
erased some comments

Kruchy1980 committed:
NIL

Kruchy1980 committed:
conflicts solving in progress

Kruchy1980 committed:
Problems solved

Kruchy1980 committed:
nil
